hoodie-bench/src/main/java/com/uber/hoodie/integrationsuite/job/action/ValidateAction.java

```diff
@@ -0,0 +1,18 @@
+package com.uber.hoodie.integrationsuite.job.action;
+
+/**
+ * Represents different validation operations to be run during the the execution of the HudiTestSuiteJob
+ */
+public class ValidateAction extends BaseAction {
+
+  private String validationClassName;
+
+  public ValidateAction(String validationClassName) {
```

Comment:
The operation and action classes can just be a ENUM IMO.. unless you want to add more methods here, it seems superflous to me.